src/test/resources/org/prebid/server/it/openrtb2/sharethrough/test-auction-sharethrough-response.json

```diff
@@ -0,0 +1,183 @@
+            "status": 200,
+            "uri": "http://localhost:8090/sharethrough-exchange?placement_key=abc123&bidId=bid&consent_required=false&consent_string=consentValue&instant_play_capable=true&stayInIframe=true&height=50&width=50&supplyId=FGMrCMMc&strVersion=1.0.0"
+          }
+        ]
+      },
+      "resolvedrequest": {
+        "id": "tid",
+        "imp": [
+          {
+            "id": "bid",
+            "banner": {
+              "format": [
+                {
+                  "h": 250,
+                  "w": 300
+                }
+              ]
+            },
+            "ext": {
+              "sharethrough": {
+                "iframe": true,
+                "iframeSize": [
+                  50,
+                  50
+                ],
+                "pkey": "abc123"
+              }
+            }
+          }
+        ],
+        "regs": {
+          "ext": {
+            "gdpr": 0
+          }
+        },
+        "cur": [
+          "USD"
+        ],
+        "device": {
+          "dnt": 2,
+          "ifa": "ifaId",
+          "ip": "192.168.244.1",
+          "language": "en",
+          "pxratio": 4.2,
+          "ua": "userAgent"
+        },
+        "site": {
+          "domain": "example.com",
+          "ext": {
+            "amp": 0
+          },
+          "page": "http://www.example.com",
+          "publisher": {
+            "id": "publisherId"
+          }
+        },
+        "source": {
+          "fd": 1,
+          "tid": "tid"
+        },
+        "test": 1,
+        "at": 1,
+        "tmax": 1000,
+        "user": {
+          "ext": {
+            "consent": "consentValue",
+            "digitrust": {
+              "id": "id",
+              "keyv": 123,
+              "pref": 0
+            }
+          }
+        },
+        "ext": {
+          "prebid": {
+            "aliases": {
+              "appnexusAlias": "appnexus",
+              "conversantAlias": "conversant"
+            },
+            "targeting": {
+              "pricegranularity": {
+                "precision": 2,
+                "ranges": [
+                  {
+                    "max": 20,
+                    "increment": 0.1
+                  }
+                ]
+              },
+              "currency": {
+                "rates": {
+                  "EUR": {
+                    "USD": 1.2406
+                  },
+                  "USD": {
+                    "EUR": 0.811
+                  }
+                }
+              },
+              "includewinners": true,
+              "includebidderkeys": true
+            },
+            "cache": {
+              "bids": {},
+              "vastxml": {
+                "ttlseconds": 120
+              }
+            }
+          }
+        }
+      }
+    },
+    "responsetimemillis": {
+      "sharethrough": "{{ sharethrough.response_time_ms }}",
+      "cache": "{{ cache.response_time_ms }}"
```

Comment:
You did replace these ones though) Please keep in mind all above-mentioned placeholders - we must always replace them to avoid using hard-coded values.